controllers/news-controllers.js

```diff
@@ -28,8 +30,28 @@ const getArticlesById = (req, res, next) => {
     })
 }
 
+const postNewComment = (req, res, next) => {
+    const {article_id} = req.params;
+    const newComment = req.body;
+    checkArticleExists(article_id)
+    .then((response) => {
+        if (response === true) {
```

Comment:
I'd maybe flip the logic here so it becomes more readable for Hoomanns! check article exists ? >> true.  should be when it DOES exist. Small thing, but helps when debugging or collaborating on this codebase.